Dockerfile

```diff
@@ -3,3 +3,11 @@ RUN apt-get update -y && apt-get install -y \
 python3-pip \
 python3-setuptools \
 libsystemd-dev
+
+WORKDIR /code
+COPY . .
```

Comment:
This command copies all files from the repo, including the .git folder (if I am not mistaken). Do we have a dockerignore file?

Reply:
Adding .dockerignore file.